_data/versions.yml

```diff
@@ -1,3 +1,3 @@
-java_sdk: 0.12.43
+java_sdk: 0.12.47
```

Comment:
What is this used for? Should it match the version in JavaSDK (which will need to be bumped to 12.49 or 12.50)?